include/lldb/Target/Process.h

```diff
@@ -1590,19 +1590,14 @@ class Process : public std::enable_shared_from_this<Process>,
   void PrintWarningOptimization(const SymbolContext &sc);
 
   //------------------------------------------------------------------
-  /// Print a user-visible warning about a module having Swift settings
-  /// incompatible with the current system
-  ///
-  /// Prints a async warning message to the user one time per Process for a
-  /// Module
-  /// whose Swift AST sections couldn't be loaded because they aren't buildable
-  /// on
+  /// Prints a async warning message to the user one time per Process for a Module
+  /// whose Swift AST sections couldn't be loaded because they aren't buildable on
   /// the current machine.
   ///
   /// @param [in] module
   ///     The affected Module.
   //------------------------------------------------------------------
-  void PrintWarningCantLoadSwift(const Module &module);
+  void PrintWarningCantLoadSwift(const Module &module, const char *details);
```

Comment:
Should this just be a `std::string` or a `StringRef` ?

Reply:
This is feeding into `Process::PrintWarning(uint64_t warning_type, const void *repeat_key, const char *fmt, ...)` so a StringRef is not an option because the NUL isn't guaranteed. A std::string would work though.